chain/x/zoracle/internal/keeper/data_source.go

```diff
@@ -13,25 +13,28 @@ func (k Keeper) SetDataSource(ctx sdk.Context, id types.DataSourceID, dataSource
 }
 
 // AddDataSource adds the given data source to the storage.
-func (k Keeper) AddDataSource(ctx sdk.Context, owner sdk.AccAddress, name string, description string, fee sdk.Coins, executable []byte) sdk.Error {
+func (k Keeper) AddDataSource(
```

Comment:
Please just add a simple test for adding data source and check data source id that return from the function is correct.

Reply:
Or you can test on handler that emits the event correctly,